### DISCUSSION TITLE
A bridge component idea !

### DISCUSSION THREAD

**Discussion by ImSquiDev** [bridge-core/bridge.#324] [Ideas]
Hi,
I want to make a block that turns on with a redstone power but i don't know how to do that, so could you add a bridge. component that execute events when the block is powered by redstone ( If it's possible , of course) please? I 'm french so sorry if my english is bad...

I hope you'll read this and thanks for this app, it's great !

**Comment by solvedDev** [comment]
This is not possible with Add-Ons just yet :)

**Comment by ImSquiDev** [comment]
Ok, thanks for your comment !
I'll publish my component idea on minecraft suggestion.  😃
